frontend/src/scenes/notebooks/Notebook/Editor.tsx

```diff
@@ -29,7 +29,6 @@ import { NotebookNodeImage } from '../Nodes/NotebookNodeImage'
 
 import { EditorFocusPosition, EditorRange, JSONContent, Node, textContent } from './utils'
 import { SlashCommandsExtension } from './SlashCommands'
-import { BacklinkCommandsExtension } from './BacklinkCommands'
```

Comment:
I'd be in favour of deleting this file if we're not using it. Or leaving it and just changing the key mapping to `char: '#',` or similarly appropriate character

Reply:
Tempted to keep it removed as it is super buggy tbh. It steals mouse focus and sometimes gets stuck in a weird state.

Reply:
Then we can come back to it later when we have the new search